auth-web/src/components/auth/staff/continuation-application/ContinuationApplicationTable.vue

```diff
@@ -97,60 +163,222 @@
 </template>
 
 <script lang="ts">
-import { defineComponent, reactive } from '@vue/composition-api'
-import BusinessServices from '@/services/business.services'
-import { ContinuationReviewIF } from '@/models/continuation-review'
+import { ContinuationReviewIF, ReviewFilterParams } from '@/models/continuation-review'
+import {
+  DEFAULT_DATA_OPTIONS,
+  cachePageInfo,
+  getAndPruneCachedPageInfo,
+  getPaginationOptions,
+  hasCachedPageInfo
+} from '@/components/datatable/resources'
+import { PropType, computed, defineComponent, reactive, toRefs, watch } from '@vue/composition-api'
+import CommonUtils from '@/util/common-util'
+import ConfigHelper from '@/util/config-helper'
+import { DataOptions } from 'vuetify'
+import { SessionStorageKeys } from '@/util/constants'
+import debounce from '@/util/debounce'
+import moment from 'moment'
+import { useI18n } from 'vue-i18n-composable'
+import { useStaffStore } from '@/stores/staff'
 
 export default defineComponent({
   name: 'ContinuationApplicationTable',
-  setup () {
+  props: {
+    reviewSessionStorageKey: {
+      type: String as PropType<SessionStorageKeys>,
+      default: SessionStorageKeys.ReviewSearchFilter
+    },
+    reviewPaginationNumberOfItemsKey: {
+      type: String as PropType<SessionStorageKeys>,
+      default: SessionStorageKeys.ReviewPaginationNumberOfItems
+    },
+    reviewPaginationOptionsKey: {
+      type: String as PropType<SessionStorageKeys>,
+      default: SessionStorageKeys.ReviewPaginationOptions
+    }
+  },
+  setup (props) {
+    const { t } = useI18n()
+    const staffStore = useStaffStore()
+    const statusDisplayMap = {
+      AWAITING_REVIEW: 'Awaiting Review',
+      CHANGE_REQUESTED: 'Change Requested',
+      RESUBMITTED: 'Resubmitted',
+      REJECTED: 'Rejected',
+      APPROVED: 'Approved',
+      ABANDONED: 'Abandoned'
+    }
+
```

Comment:
The Quality Gate failed warning should not be here anymore.

Reply:
Yes, I fixed it. See previous comment.